### PR TITLE
updated packages

### PR DESCRIPTION
**Pull request by coykwan**
- [x] Related GitHub issue(s) linked in PR description
- [ ] Destination branch merged, built and tested with your changes
- [ ] Code formatted and follows best practices and patterns
- [ ] Code builds cleanly (no _additional_ warnings or errors)
- [ ] Manually tested
- [ ] Automated tests are passing
- [ ] No _decreases_ in automated test coverage
- [ ] Documentation updated (readme, docs, comments, etc.)
- [ ] Localization: No hard-coded error messages in code files (_minimally_ in string constants)
- [ ] New component and/ or properties? Make sure to update storybook

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,13 +28,13 @@
...
         "andculturecode-javascript-react": "0.1.0",
-        "andculturecode-scss-grid": "2.0.4",
+        "andculturecode-scss-grid": "3.0.0",
         "axios": "0.19.2",
         "bourbon": "6.0.0",
         "i18next": "19.4.5",
         "immutable": "4.0.0-rc.12",
-        "react": "16.13.1",
-        "react-dom": "16.13.1",
+        "react": "^16",
```
Comment (andcultbrisbin) [inline package.json:36]: Let's continue specifying the minor and patch versions. 16.13.1

File: package.json [hunk 2]
```diff
@@ -28,13 +28,13 @@
...
+        "react-dom": "^16",
```
Comment (andcultbrisbin) [inline package.json:37]: Let's continue specifying the minor and patch versions. 16.13.1

**Review by andcultbrisbin — CHANGES_REQUESTED** [review]
This looks good to me! Let's adjust the react/react-dom in the package json.

**Review by myty — APPROVED** [review]
Looks good

**Pull request merged**